src/_tutorials/server/get-started.md

```diff
@@ -1,263 +1,237 @@
 ---
-title: Get started with server-side Dart
-description: Get Dart and run a Dart app.
+title: "Get started: Dart native apps"
```

Comment:
Dart native or Dart Native?

/cc @messi01

Reply:
@kwalrath what's the context for this? Where is this tutorial on the current dartlang site? Have we historically used "Dart native"?

Reply:
@messi01 I was just writing you email... I hope to get it to you within the hour!

Reply:
@mit-mit, I thought Flutter counted as Dart native? I'm getting confused...

Reply:
> @mit-mit, I thought Flutter counted as Dart native? I'm getting confused...

Flutter apps are powered by Dart Native, that right. Can you clarify what the confusion is?

Reply:
> I'd love some more context on this! Where is this tutorial on the current dartlang site?

This doesn't currently exists. Historically we have only had the Dart VM (the `dart` command). The two new things in this PR we are adding are Dart AOT (the `dart2aot` command), and debugging using Dart DevTools.